### ISSUE TITLE
Remove SI support from list of officially supported LFVM configurations

### ISSUE THREAD

**Issue by HerbertJordan** [Fantom-foundation/Tosca#763]
For backward-compatibility the `lfvm-si` configuration is currently listed as a non-experimental configuration supported for productive use cases. This is wrong and needs to be fixed.

In order to do so, Aida needs to be updated to load experimental configurations. Once this is done and merged into Aida's main branch, the `lfvm-si` configuration can be removed.
